Missing initializer for PROCESSENTRY32 type

This looks funky.

```bash
 g++ -c -fno-keep-inline-dllexport -O2 -std=gnu++11 -pthread -ffunction-sections -fdata-sections -Wall -Wextra -Wextra -fexceptions -mthreads -DUNICODE -D_UNICODE -DWIN32 -DMINGW_HAS_SECURE_API=1 -DQT_OPENSOURCE -DWAIRTOHUB -D__RT_AUDIO__ -DUSE_WEAK_JACK -D__WIN_32__ -D_WIN32_WINNT=0x0600 -DWIN32_LEAN_AND_MEAN -D__WINDOWS_ASIO__ -D__WINDOWS_WASAPI__ -DQT_NO_DEBUG -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_NETWORK_LIB -DQT_CORE_LIB -I../../jacktrip -I. -Ifaust-src-lair/stk -I"C:/Program Files/JACK2/include" -I../externals/weakjack -Ifaust-src-lair -IC:/vcpkg/installed/x64-mingw-static/include/rtaudio -I../../qt-static/include -I../../qt-static/include/QtWidgets -I../../qt-static/include/QtGui -I../../qt-static/include/QtNetwork -I../../qt-static/include/QtCore -Irelease -I. -I../../qt-static/mkspecs/win32-g++  -o release/main.o ../src/main.cpp
../src/main.cpp: In function 'bool isRunFromCmd()':
../src/main.cpp:168:27: warning: missing initializer for member 'tagPROCESSENTRY32W::cntUsage' [-Wmissing-field-initializers]
     PROCESSENTRY32 pe = {0};
                           ^
../src/main.cpp:168:27: warning: missing initializer for member 'tagPROCESSENTRY32W::th32ProcessID' [-Wmissing-field-initializers]
../src/main.cpp:168:27: warning: missing initializer for member 'tagPROCESSENTRY32W::th32DefaultHeapID' [-Wmissing-field-initializers]
../src/main.cpp:168:27: warning: missing initializer for member 'tagPROCESSENTRY32W::th32ModuleID' [-Wmissing-field-initializers]
../src/main.cpp:168:27: warning: missing initializer for member 'tagPROCESSENTRY32W::cntThreads' [-Wmissing-field-initializers]
../src/main.cpp:168:27: warning: missing initializer for member 'tagPROCESSENTRY32W::th32ParentProcessID' [-Wmissing-field-initializers]
../src/main.cpp:168:27: warning: missing initializer for member 'tagPROCESSENTRY32W::pcPriClassBase' [-Wmissing-field-initializers]
../src/main.cpp:168:27: warning: missing initializer for member 'tagPROCESSENTRY32W::dwFlags' [-Wmissing-field-initializers]
../src/main.cpp:168:27: warning: missing initializer for member 'tagPROCESSENTRY32W::szExeFile' [-Wmissing-field-initializers]
```

--- a/src/main.cpp
+++ b/src/main.cpp
@@ -165,7 +165,8 @@ BOOL WINAPI windowsCtrlHandler(DWORD fdwCtrlType)
 bool isRunFromCmd() {
     //Get our parent process pid
     HANDLE h = NULL;
-    PROCESSENTRY32 pe = {0};
+    PROCESSENTRY32 pe;
+    ZeroMemory(&pe, sizeof(PROCESSENTRY32));
     DWORD pid = GetCurrentProcessId();
     DWORD ppid = 0;
     pe.dwSize = sizeof(PROCESSENTRY32);
@@ -180,7 +181,7 @@ bool isRunFromCmd() {
         } while (Process32Next(h, &pe));
     }
     CloseHandle(h);
-    
+
     //Get the name of our parent process;
     char pname[MAX_PATH] = {0};
     DWORD size = MAX_PATH;
@@ -189,7 +190,7 @@ bool isRunFromCmd() {
     if (h) {
         if (QueryFullProcessImageNameA(h, 0, pname, &size)) {
             CloseHandle(h);
-            
+
             //Check if our parent process is a command line.
             if (size >= 14 && strncmp(pname + size - 14, "powershell.exe", 14) == 0) {
                 return true;
@@ -201,7 +202,7 @@ bool isRunFromCmd() {
             CloseHandle(h);
         }
     }
-    
+
     return false;
 }
 #endif
@@ -222,7 +223,7 @@ int main(int argc, char* argv[])
         }
 #endif  // _WIN32
         app->setApplicationName("QJackTrip");
-        
+
         QCommandLineParser parser;
         QCommandLineOption verboseOption(QStringList() << "V" << "verbose");   
         parser.addOption(verboseOption);
@@ -230,7 +231,7 @@ int main(int argc, char* argv[])
         if (parser.isSet(verboseOption)) {
             gVerboseFlag = true;
         }
-        
+
         window.reset(new QJackTrip);
         window->setArgc(argc);
         QObject::connect(window.data(), &QJackTrip::signalExit, app.data(),

